default/generated/eap8/eap_channel_manifest_8_0_upgrade.mta.yaml

```diff
@@ -0,0 +1,12 @@
+- ruleID: eap_channel_manifest_8_0_upgrade-00001
+  description: "The JBoss EAP Channel Manifest 8.0 should be upgraded."
+  category: mandatory
+  effort: 1
+  links:
+    - url: https://docs.redhat.com/en/documentation/red_hat_jboss_enterprise_application_platform/8.1/html/migration_guide/migrate-a-jboss-eap-application-s-maven-project-to-jboss-eap-8-1_default
```

Comment:
Let's comment these out and leave a `TODO` to activate them when they are available. Applicable for every other broken link.